src/collection-preferences/__integ__/pages/collection-preferences-page.ts

```diff
@@ -1,20 +1,25 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
 
 import { CollectionPreferencesWrapper } from '../../../../lib/components/test-utils/selectors';
+import DndPageObject from '../../content-display/__integ__/pages/dnd-page-object';
 
-export const collectionPreferencesPageMixin = (Base: BasePageObject & any) =>
```

Comment:
I don't see any other use case for this function, Just wondering what was the initial reason of using it in a mixin format.

Reply:
Maybe it was used as a wrapper for both BasePageObject and DndPageObject or just a code style